lib/kernel/test/code_SUITE.erl

```diff
@@ -1447,20 +1449,61 @@ on_load_binary(_) ->
 
     {Pid1,Ref1} = spawn_monitor(fun() ->
 					code:load_binary(Mod, File, Bin),
-					true = on_load_binary:ok()
+					true = Mod:ok()
 				end),
     receive {Mod,OnLoadPid} -> ok end,
     {Pid2,Ref2} = spawn_monitor(fun() ->
-					true = on_load_binary:ok()
+					true = Mod:ok()
```

Comment:
This does not change the behaviour of the test. It is simply to make this test share more lines with the test below. :)